rust_src/src/lisp.rs

```diff
@@ -72,6 +72,10 @@ impl LispObject {
     pub fn from_float(v: EmacsDouble) -> Self {
         unsafe { make_float(v) }
     }
+
+    pub fn fallthrough() -> Self {
```

Comment:
`fallthrough` has been misinterpreted. This is a bit of C idiom. The intent is to continue into the next `switch` branch.

```
switch(value) {
    case FOO: {
        do stuff....
        break; // stop processing here
    }
    case BAR: {
        do stuff....
        FALLTHROUGH;
    }
    case THING: {
        some stuff.....
        break;
    }
}
```

If the `BAR` condition is met both the BAR code and the THING code are run. `FALLTHROUGH` is a meaningless no-op value that allows the developer to visually see that the intent is to fall through. Without it, one has to spot the CASE branches missing their `break` statements.

I suspect this is where your bugs are happening, the extra code is not running.

Reply:
Oops!!
Thanks a lot :)